<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1204</width>
    <height>609</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_superior">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>37</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>37</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">

QFrame {
background-color: #14131B;
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="btn_menu">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	border: 20px;
    padding-left: 5px; 
}

QPushButton:hover {
    background-color: #026ec1;
}

QPushButton:pressed {
    background-color: #015185;
}</string>
         </property>
         <property name="text">
          <string>  MENU </string>
         </property>
         <property name="icon">
          <iconset resource="icon.qrc">
           <normaloff>:/img/icon_light/menu.svg</normaloff>:/img/icon_light/menu.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>489</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_inferior">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignLeft">
        <widget class="QFrame" name="frame_lateral">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">* {
   border: none;
}

QPushButton {
    font: 75 11pt 'Arial';
    font-weight: Normal;    
	padding-left: 8px;
    padding-right: 10px;
    text-align: left;
    border-left: 3px solid transparent;
}

QPushButton:hover {
	background-color: #026ec1;
}


QFrame {
background-color: #14131B;
}
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="bt_0">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>38</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::PreventContextMenu</enum>
            </property>
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>  Opc_0</string>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/img/icon_light/home.svg</normaloff>:/img/icon_light/home.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="autoRepeatDelay">
             <number>300</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_1">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>38</height>
             </size>
            </property>
            <property name="text">
             <string>  Opc_1</string>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/img/icon_light/user-plus.svg</normaloff>:/img/icon_light/user-plus.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>38</height>
             </size>
            </property>
            <property name="text">
             <string>  Opc_2</string>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/img/icon_light/edit-3.svg</normaloff>:/img/icon_light/edit-3.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>38</height>
             </size>
            </property>
            <property name="text">
             <string>  Opc_3</string>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/img/icon_light/file-plus.svg</normaloff>:/img/icon_light/file-plus.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>108</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="bt_4">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>38</height>
             </size>
            </property>
            <property name="text">
             <string>  Opc_4</string>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/img/icon_light/settings.svg</normaloff>:/img/icon_light/settings.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_5">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>38</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>  Opc_5</string>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/img/icon_light/alert-circle.svg</normaloff>:/img/icon_light/alert-circle.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="main_body">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(27, 27, 35);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <widget class="QWidget" name="page_0">
             <property name="styleSheet">
              <string notr="true">*{border:none
}  

QFrame{
background-color: #1B1B23;


 
}  

</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Page_0</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_2">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Page_2</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_3">
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Page_3</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_4">
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Page_4</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_5">
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Page_5</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_1">
             <property name="styleSheet">
              <string notr="true">*{border:none}  

QFrame{
background-color: #1B1B23; 
}  </string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_1">
                <property name="text">
                 <string>Page_1</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
